### DISCUSSION TITLE
Aeolian-Skinner ODF anyone?

### DISCUSSION THREAD

**Discussion by emseedee** [GrandOrgue/grandorgue#852] [General]
Admin - not sure if this belongs here; please move it to a better place if one exists!

Does anyone know of a GO ODF for the Church of the Redeemer Aeolian-Skinner sample set? I've got the HW sample set, which isn't encrypted and doesn't seem to have any copyright notices anywhere within it, and I'd love to try it out on GO. 

If necessary, I might write my own ODF, and I'm particularly interested in how the various samples are used. There seem to be 2 HW organ definitions, both containing samples for all the pipes, one called "near" and the other "far". I'm guessing that's to do with the microphone position, but I don't understand how to use them. Do I just choose the one I like the best?

Thanks for reading

Mike

**Comment by ahall41** [comment]
This is available on [MPS Orgelseite](https://mps-orgelseite.de/home/filebase/). Only trouble is (for the surround version), the link to the Carillon no longer works.
